icpc-wf/ansible/common_tasks_all.yml

```diff
@@ -94,17 +105,6 @@
         mode: 0600
       when: DJ_GIT_SSH_KEY is defined
 
-    - name: Create SSH config
-      template: src=files/ssh-config-domjudge.j2 dest=/home/domjudge/.ssh/config owner=domjudge group=domjudge mode=0600
-      when: DJ_GIT_HOST is defined
-
-    - name: add all hosts to hosts file
-      lineinfile:
-        dest: /etc/hosts
-        regexp: '{{ item }}$'
-        line: "{{ hostvars[item].ansible_host }}	{{ item }}"
-      loop: "{{ groups['all'] }}"
```

Comment:
Why do we remove this block?

Reply:
Because I have 2 branches mixed which shouldn't be mixed.